packages/test-cases/cases/bind/sanity/output/wasm-as/query/env.ts

```diff
@@ -3,3 +3,8 @@ import {
 } from "./QueryEnv";
 
 export let env: QueryEnv | null = null;
+
+export function ensureEnv(): QueryEnv {
```

Comment:
ensureEnv is a bit unclear
maybe ensureEnvDefined ?

Reply:
I find it too verbose though! open for other suggestions :)

Reply:
requireEnv? That's a common pattern

Reply:
sounds good